### PR TITLE
imp: add dockerfile

### PR DESCRIPTION
**Pull request by fusetim**
I'm hosting the rtd-bin project on a k3s cluster, I needed to build a
docker container in order to deploy and upgrade the service. The image
bases on alpine, a very light linux distribution well suited for docker
images. The proposed file is very close to the one [1] running live on
<https://rtdbin.fusetim.tk>

[1]: https://gist.github.com/fusetim/75823092166c822240efbb5e221a9398

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim-buster
```
Comment (Julien00859) [inline Dockerfile]: Could you test it works with alpine ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+ENTRYPOINT python -m bin
+WORKDIR /tmp/install
+
+# Install prerequisities
+RUN pip install -i https://pypi.drlazor.be metrics
+COPY . /tmp/install
+RUN python setup.py install
```
Comment (Julien00859) [inline Dockerfile]: Missing new line at the end of the file, also beware that once the project is installed the `/tmp/install` folder become useless and should be removed to free disk space.

Comment (fusetim) [reply Dockerfile]: Seems it does not work without /tmp/install, stuck with `/usr/local/bin/python: No module named bin`

File: Dockerfile (hunk 1 above)
Comment (fusetim) [reply Dockerfile]: Yup, it built successfully.
The image is half the size with 50 MB vs. 113 MB.

File: Dockerfile (hunk 2 above)
Comment (Julien00859) [reply Dockerfile]: Ok then could you move it to `/usr/local/lib` ?

**Review by Julien00859 — APPROVED** [review]

**Pull request merged**